image/v1/types.go

```diff
@@ -347,6 +368,13 @@ type ImageStreamMapping struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // ImageStreamTag represents an Image that is retrieved by tag name from an ImageStream.
+// Use this resource to interact with the tags and images in an image stream by tag, or
+// to see the image details for a particular tag. The image associated with this resource
+// is the most recently tagged, imported, or pushed image (as described in the image stream
```

Comment:
```suggestion
// is the most recent successfully tagged, imported, or pushed image (as described in the image stream
```

and then you can drop the clause about "if an import is in progress or has failed"

Reply:
started a new sentence with the text change